src/archetypes/java-web-app/deploy.ts

```diff
@@ -0,0 +1,39 @@
+const BasicJavaApplicationDeployer = require.main?.exports.BasicJavaApplicationDeployer as any
+const path = require('path')
+
```

Comment:
"${NAME}${SUFFIX}-webade-jar" is not being used for deployment. needs to be removed.